components/team-info/sections/DocumentsSection.vue

```diff
@@ -84,7 +84,7 @@ const menuOptions: MenuOption[] = [
     label: 'Retrospectives',
     key: 'retrospectives',
     children: documents.value
-      ?.filter((doc) => doc.title?.startsWith('retrospective'))
+      ?.filter((doc) => doc.title?.startsWith('r'))
```

Comment:
Why?

Reply:
to iste ako s minute books

Reply:
nechaj tak ako bolo, nedávaj iba s.. do title v summary.md daj "retrospective-summary.pdf" a samotný súbor premenuj na "retrospective-summary.md"